source/fundamentals/monitoring/cluster-monitoring.txt

```diff
@@ -85,7 +85,8 @@ The following sections show sample output for each type of SDAM event.
 serverDescriptionChanged
 ^^^^^^^^^^^^^^^^^^^^^^^^
 
-.. code-block:: js
+.. code-block:: json
```

Comment:
I don't think these are JSON, but rather Node objects being printed out.
One way you can tell is that "ServerDescriptionChangedEvent" isn't a key or part of a value. Everything in JSON needs to be a key or part of a value and paired with a ":". 

The pygments lexer (what parses the code block and guides the code that adds the highlighting) doesn't recognize this as JSON and therefore does not have any highlighting. The current version in which Javascript is identified as the code block language has highlighting.

Could you change all instances in which the output represents some output that isn't pure JSON back to Javascript so that it maintains the highlighting?

Note that as long as each preceding line has valid JSON, syntax highlighting continues to work; i.e. when you have JSON objects on separate lines not joined by commas, it still renders correctly.